library/src/main/java/com/mikepenz/materialdrawer/model/ExpandableDrawerItem.java

```diff
@@ -72,7 +87,7 @@ public void bindView(ViewHolder viewHolder) {
         bindViewHelper(viewHolder);
 
         //make sure all animations are stopped
-        viewHolder.arrow.setColor(getIconColor(ctx));
+        viewHolder.arrow.setColor(this.arrowColor != null ? this.arrowColor.getColorInt() : getIconColor(ctx));
```

Comment:
you should use the `color` method from the `ColorHolder` as the above used method will just return the colorInt. 
https://github.com/mikepenz/Materialize/blob/develop/library/src/main/java/com/mikepenz/materialize/holder/ColorHolder.java#L123
(0 is returned if no value was set)